internal/migrations/migrate_visualization_test.go

```diff
@@ -0,0 +1,60 @@
+package migrations
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/openshift/cluster-logging-operator/apis/logging/v1"
+)
+
+var _ = Describe("Migrating ClusterLogging instance", func() {
+
+	Context("when migrating forwarder and collection.logs to collection", func() {
+		It("should return clusterlogging as-is when collection is not defined", func() {
```

Comment:
What is the relevance of the collectionspec in relation to visualization.  I don't see the dependency